<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NSDrawTextEditFormatDialog</class>
 <widget class="QDialog" name="NSDrawTextEditFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文本格式</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>116</y>
     <width>321</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>时间格式</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>204</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOk">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="spinBoxMaxLength">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>28</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>54</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>小数位数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>字符串最大长度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDateFormat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDecimal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
